tmt/steps/provision/__init__.py

```diff
@@ -8,36 +8,57 @@
 from click import echo
 
 from tmt.utils import SpecificationError
+from tmt.steps.provision import vagrant, localhost
 
 
 class Provision(tmt.steps.Step):
     """ Provision step """
-
-    # Default implementation for provision is a virtual machine
+    name = 'provision'
+
+    # supported provisioners are not loaded automatically, import them and map them in how_map
+    how_map = {
```

Comment:
How about a missing one from `how_map`? That should simply go to `ProvisionVagrant` as one can create their own `Providers` (vagrant calls them that) and they should simply run [without any modifications in ProvisionVagrant](https://github.com/psss/tmt/blob/master/tmt/steps/provision/vagrant.py#L275).

Reply:
We can try it :)